### PR TITLE
BLOCKS-332 CSS Class Duplicates/Conflicts

### PR DESCRIPTION
**Pull request by greilberger13**
Change name of "catro-tabs" to "catblocks-tabs" and "search" to "catblocks-search", but not sure about search.

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/library/scss/share.scss [hunk 1]
```diff
@@ -98,6 +98,6 @@
   }
 }
 
-.search {
+.catblocks-search {
```
Comment (MichaelFlucher) [inline src/library/scss/share.scss:101]: If you change this class name here, where is the belonging change of the name inside the JS Code?

**Review by MichaelFlucher — CHANGES_REQUESTED** [review]
Please change the `.advanced-theme` also

File: src/library/scss/share.scss (hunk 1 above)
Comment (greilberger13) [reply src/library/scss/share.scss:101]: Sorry for that one - did not found it in the first place and was not sure if it is even used.
What is the next step, if everything is done and working again?

Comment (MichaelFlucher) [reply src/library/scss/share.scss:101]: Simply amend your commit and force push, then I will review again

Comment (greilberger13) [reply src/library/scss/share.scss:101]: Done. Sorry again for the inconvenience. Hope this is correct. 
I am not sure about the `.catblocks-search` class in `_blockly.scss`. (I changed it too)

**Pull request closed without merging**